autotst/calculator/gaussian.py

```diff
@@ -176,11 +180,16 @@ def get_conformer_calc(self):
         - torsion (Torsion): A `Torsion` object that you want to perform hindered rotor calculations about
         - settings (dict): a dictionary of settings containing method, basis, mem, nprocshared
         - scratch (str): a directory where you want log files to be written to
+        - steps (int): the number of steps you want performed in this scan
```

Comment:
Why are steps and step_size in the comments of this method?

Reply:
This should be in rotor_calc, not conformer calc.  I removed steps and see_size in commit bb2d188